### ISSUE TITLE
Add option to enable/disable colored output with the Pretty printer

### ISSUE THREAD

**Issue by metmajer** [segmentio/terraform-docs#78]
### Prerequisites

Put an `x` into the box that applies:

- [ ] This issue describes a bug.
- [x] This issue describes a feature request.

For more information, see the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

Currently, the *pretty printer* in `internal/pkg/pretty` generates colored output on stdout by default. We would like to have the user enable colors explicitly using `--with-colors` (otherwise print no colors).

**Comment by metmajer** [comment]
@jocgir

**Comment by beatcracker** [comment]
Btw, colored output doesn't work on Windows. Even if run in Windows 10 `cmd` which should support  escape sequences. So maybe it should be disabled on Win by default until (and if) we fix it.

**Comment by metmajer** [comment]
Thanks for your input, @beatcracker! Thinking about removing colors entirely...

Aside, I have been wondering if there's a use-case for the pretty printer. Has anyone got one to share?

**Comment by metmajer** [comment]
@beatcracker @jocgir @khos2ow @sjauld @X-Guardian

All, asking for your feedback on this one. I, myself, don't see value in the pretty printer output and have been thinking of deprecating it entirely before arriving at v.1.0.0. Options are:

1) Leave the pretty printer as is
2) Leave the pretty printer and remove non-portable color codes
3) Remove the pretty printer and use `json` as default
4) Remove the pretty printer and use `markdown` as default
5) Remove the pretty printer and make the printer a required argument (no default)

Really not a high-prio thing, but let's share thoughts. Cheers!

**Comment by sjauld** [comment]
Tbh I'm not even sure what the pretty printer is. I only use markdown. I
vote #4!

**Comment by sjauld** [comment]
If no one is using it, it makes sense to drop it. Every output format means
more tests to write. Maybe it was originally an internal segmentio thing.

This tool is designed to build documentation, so markdown seems the most
sensible default out of the currently available outputs.

**Comment by metmajer** [comment]
Thanks for sharing thoughts, @sjauld!

**Comment by X-Guardian** [comment]
I've never used the pretty print, and can't think I ever would. If you don't want to maintain the code, options 3 or 4 seem sensible, as iI think it should return some output by default.

**Comment by khos2ow** [comment]
I've used pretty print may be handful of times when I was in a completely new module/config that I was asked to participate on. Immediately in the terminal I was able to get the `variables` I need to care about.

TBH I don't mind keeping the feature around or may be even switch the default behaviour to non-colored, only if it made sense in terms of time and effort.

Otherwise I think option 3 would be better since `markdown` has two subcommands (`table` and `documents`) albeit it has its own default (`table`).
